src/chocolatey/infrastructure.app/builders/ConfigurationBuilder.cs

```diff
@@ -355,6 +355,9 @@ private static void set_global_options(IList<string> args, ChocolateyConfigurati
                         .Add("trace",
                              "Trace - Show trace messaging. Very, very verbose trace messaging. Avoid except when needing super low-level .NET Framework debugging. Available in 0.10.4+.",
                              option => config.Trace = option != null)
+                        .Add("version",
+                             "Version - Prints out the Chocolatey version.",
+                             option => {})

```

Comment:
IIRC this is may conflict with install / upgrade / uninstall / search as is. It would be worth checking if the `version` of those just nicely overrides this or not.